bindata/bootkube/manifests/00_etcd-endpoints-cm.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: etcd-endpoints
+  namespace: openshift-etcd
+  annotations:
+    alpha.installer.openshift.io/etcd-bootstrap: {{ .BootstrapIP }}
```

Comment:
Something I think slipped by in all the previous noise was that I had deleted the endpoints resource from the manifests list, which would have broken the current version of the apiservers until they've been updated to read the configmap. I fixed that here.